webapp/src/css/inbox.less

```diff
@@ -108,6 +109,53 @@ body {
   display: none;
 }
 
+#privacy-policy-wrapper {
+  z-index: 11;
+  position: absolute;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  right: 0;
+  overflow: hidden;
+  background: white;
+
+  .container {
+    height: 100%;
+
+    .loader {
+      top: 50%;
+      margin-top: -20px;
+    }
+
+    .content {
+      height: 100%;
+      padding-bottom: 67px;
+      position: relative;
+
+      .html-content {
+        height: 100%;
+        overflow-y: auto;
+      }
+
+      .footer {
+        position: absolute;
+        border-top: 1px solid @gray-light;
+        padding: 15px;
+        width: 100%;
+        bottom: 0;
+
+        .btn.disabled:before {
+          content: "\f110";
+          font-family: FontAwesome;
+          display: inline-block;
+          padding-right: 10px;
+        }
```

Comment:
Is this the spinner when clicking "accept"? If so, can you use the standard `.loader.inline.small` or some combination thereof?

Reply:
Updated and pulled the spinner outside of the button.